src/scripts/main.js

```diff
@@ -354,7 +354,32 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function addInfo(arr) {
+  for (const person of arr) {
+    person.age = person.died - person.born;
+    person.century = Math.ceil(person.died / 100);
+  }
 
-// write your code here
+  return arr;
+}
+
+function addRow(array) {
+  const table = document.querySelector('.dashboard');
+
+  for (const human of array) {
+    const row = table.createElement('tr');
```

Comment:
The method `createElement` should be called on the `document` object, not `table`. Change `table.createElement('tr')` to `document.createElement('tr')`.